code-challenges.js

```diff
@@ -10,10 +21,20 @@
 
 var fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
 // Expected output: [-9, 9, 7, 199]
+
 var fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
 // Expected output: [-823, 7, 23]
 
 
+const onlyOdd = (array) => {
+   let newArr = array.filter(value => typeof value === "number").filter(value => value % 2).sort((a,b) => a - b)
```

Comment:
great! just be careful with `value => value % 2` on its own as a conditional. if there is any remainder after a number is divided by 2, it would be odd. with this logic, 1 _and_ -1 are technically "truthy" values returned from `value => value % 2`, which allows your code to continue as expected. however, -1 == true will actually not work. 

took the long way but the point is, to make your code a little more robust and to avoid some bugs you can add a little more logic to you conditional by saying `(value => value % 2 === 1 || value => value % 2 === -1)`

does that make sense?